.github/workflows/CI-unixish.yml

```diff
@@ -465,6 +465,9 @@ jobs:
           # rules 5.4 and 5.5 which have standard-dependent options.
           ../../cppcheck --dump -DDUMMY --suppress=uninitvar --inline-suppr misra/misra-test.c --std=c89 --platform=unix64 
           python3 ../misra.py -verify misra/misra-test.c.dump
+          # Test slight MISRA differences in C11 standard
+          ../../cppcheck --dump -DDUMMY --suppress=uninitvar --inline-suppr misra/misra-test-c11.c --std=c11 --platform=unix64
+          python3 ../misra.py -verify misra/misra-test-c11.c.dump
```

Comment:
I added this new test because I didn't see a good way to test this change with the existing test framework.

If you'd rather see this tested any other way, I'm happy to change the approach.

Reply:
this is fine :+1: